nextflow.config

```diff
@@ -264,8 +264,7 @@ validation {
     failUnrecognisedParams = false
     lenientMode            = false
     showHidden             = false
-    defaultIgnoreParams    = 'genomes,igenomes_base,monochromeLogs,fasta,fasta_fai,bwa,dict'
-    defaultIgnoreParams = ["genomes"]
+    defaultIgnoreParams    = ['genomes', 'modules_testdata_base_path']
```

Comment:
should this go in `tests/nextflow.confg` or `nf-test.config` as not to have test config in the main config?

Reply:
I'm not sure without digging in where it is actually used, I just saw it was being flagged as unknown parameter when I ran the test profile.

We can put it into the nextflow config as a (hidden) parameter, we can leave it here in the parameters to ignore or we can try and remove it.

Reply:
It's probably not worth it, but if you want to, it looks like others put it in the test config:
https://github.com/nf-core/rnafusion/blob/f27e73205bff5a8ccbc367891768db6d6aaada37/tests/nextflow.config#L9

Perhaps an issue for another day.

Reply:
Yeah, that is where it actually is in fastquorum at the moment. But that means when you run the test profile, nf-schema complains that it doesn't know what modules_testdata_base_path is.